### PR TITLE
CLC-5996, front-end wiring for 'Welcome Delegate' page and widget to link accounts

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-5996
https://jira.ets.berkeley.edu/jira/browse/CLC-5973

The magic URL would be /welcome_delegate The next step is to put some meat on the bones of widgets/delegated_access/link_accounts.html  Plus, 'cs_delegated_access ' feature flag.

(See relevant comments on closed PR #4490)

### CODE REVIEW

**Comment by johncrossman** [comment]
Travis is 50% green

**Comment by paulkerschen** [comment]
Our Story Begins.

File: src/assets/templates/welcome_delegate.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="cc-page-welcome-delegate">
```
Comment (christianvuerings) [inline src/assets/templates/welcome_delegate.html]: @johncrossman would it be possible to put the `ng-if` on here instead? That way none of the html below should be loaded.

`data-ng-if="api.user.profile.features.cs_delegated_access"`

Comment (johncrossman) [inline src/assets/templates/welcome_delegate.html]: @christianvuerings - I will make that change in upcoming PR w/ widget work. Thanks

**Pull request merged**